docs/unified/novacustom/recovery.md

```diff
@@ -81,102 +141,287 @@ make BOARD=arduino/mega2560
 make BOARD=arduino/mega2560 flash
 ```
 
-#### Flashing
+### Recovery steps
 
-- Unscrew the bottom cover from the laptop
+=== "everything else"
 
-=== "NS5x / NS7x"
-    ![](/images/ns50mu_board_chips.jpg)
+=== "V560"
 
-=== "NV4x"
-    ![](/images/nv4x_board_chips.jpg)
+    1. For the V560 laptops you will need to remove two stripes that are
+       holding keyboard in place.
 
-- Disconnect the battery (1)
+        ![](/images/v560tape1.webp)
 
-> All power must be removed from the laptop during flashing.
+    1. Second stripe is located under the heat sink.
 
-  ![](/images/nvc_ec_flash/ns5x_battery_unplugged.jpg)
+        ![](/images/v560heatsink.webp)
+        ![](/images/v560tape2.webp)
 
-- Reattach the bottom cover (without screwing it in) and flip the laptop over
-- Put the laptop on its side, while the screen is in an opened state. By using
-  a little screw driver, carefully push in the screw holes of the keyboard and
-  take the keyboard out. Please check out
-  [this video](https://www.youtube.com/watch?v=A66cLJcMLbI) for a demonstration.
+1. Remove the two screws holding the keyboard in place. They are indicated on the
```

Comment:
When V560 is selected, list numbering restarts from 1 here. Maybe add a heading above this, to turn it into another section.

Something like a `Model specific preparation` section with tabs for V560 and other models